Example/SPGDPRExampleAppUITests/SPGDPRExampleAppUITests.swift

```diff
@@ -20,59 +20,143 @@ class SPGDPRExampleAppUITests: QuickSpec {
             self.app = ExampleApp(XCUIApplication())
             self.app.launch()
             self.app.clearConsentButton.tap()
+            Nimble.AsyncDefaults.Timeout = 10
+            Nimble.AsyncDefaults.PollInterval = 1
         }
 
-        afterSuite {
+        afterEach {
             self.app.clearConsentButton.tap()
             /// This is a side effect. We need to "force" the UserDefaults to synchronise otherwise
             /// it won't store its in-memory values on the file system in time before closing the app.
             UserDefaults.standard.synchronize()
+            Nimble.AsyncDefaults.Timeout = 1
+            Nimble.AsyncDefaults.PollInterval = 0.01
         }
 
         describe("SPGDPRExampleAppUITests") {
-            context("when launching the app for the first time") {
-                it("1. should see the consent message") {
-                    expect(self.app.consentMessage).to(showUp())
+            it("1. Accept all through message") {
+                expect(self.app.consentMessage).to(showUp())
+                if self.app.consentMessage.exists {
```

Comment:
@vilasma is it necessary to call `swipeUp()`? The tests seem to pass without it. 
If we have to use it, do you see a way to "hide" them inside the `ExampleApp` class?